src/main/scala/viper/silver/ast/Expression.scala

```diff
@@ -446,6 +446,8 @@ case class FieldAccess(rcv: Exp, field: Field)
   def getArgs: Seq[Exp] = Seq(rcv)
   def withArgs(args: Seq[Exp]): FieldAccess = copy(rcv = args.head, field)(pos, info, errT)
 //  def asManifestation: Exp = this
+
+  override def name: String = field.name
```

Comment:
Where is this used? I didn't find the code that uses it and it's not obvious to me why it's needed here.